modules/openapi-generator/src/main/resources/scala-sttp-client/apiInvoker.mustache

```diff
@@ -0,0 +1,54 @@
+{{>licenseInfo}}
+package {{{mainPackage}}}.core
+
+import java.io.File
+
+import org.joda.time.DateTime
+import org.joda.time.format.ISODateTimeFormat
+import org.json4s.JsonAST.JString
+import org.json4s._
+import sttp.client._
+import org.openapitools.client.api.EnumsSerializers
+import org.openapitools.client.core.ApiInvoker.DateTimeSerializer
+import sttp.client.json4s.SttpJson4sApi
+import sttp.client.monad.MonadError
+
+class SttpSerializer(implicit val format: Formats = DefaultFormats ++ EnumsSerializers.all + DateTimeSerializer,
+                     implicit val serialization: org.json4s.Serialization = org.json4s.jackson.Serialization) extends SttpJson4sApi
```

Comment:
That under discussion in https://github.com/softwaremill/sttp-model/issues/5 
`json4s`library by default is a good compromise as it has AST model and supports `jackson` serialization as well. It is also possible to generate `circle` (or other supported by `sttp`) `SttpSerializer` by `additionalProperties` option instead of `json4s`. 
Make it fully library agnostic now seems bit complicated as `sttp` serializers haven't common interface.
could i ask for example regarding openApi spec for json agnostic question? generally `asJson` can be applied only to `application/json` responses, for `binary` (such as `protobuf`) solution seems too broad. will be great if you could create PR for that